### PR TITLE
Adding support of using CCComRender as parent for attached component

### PR DESCRIPTION
**Pull request by chengstory**

### CODE REVIEW

File: cocos2dx/support/component/CCComponent.h [hunk 1]
```diff
@@ -62,11 +62,14 @@ class CC_DLL CCComponent : public CCObject
     
     void setOwner(CCNode *pOwner);
     CCNode* getOwner() const;
+
+    bool isRender() const;
```
Comment (dumganhar) [inline cocos2dx/support/component/CCComponent.h]: -> isRenderer? `Render` is a `verb`

Comment (chengstory) [inline cocos2dx/support/component/CCComponent.h]: ok

File: extensions/CocoStudio/Reader/SceneReader.h [hunk 2]
```diff
@@ -42,18 +42,39 @@
 class CC_EX_DLL SceneReader
 {
 public:
-	SceneReader(void);
-	virtual ~SceneReader(void);
+    enum class AttachComponentType
```
Comment (dumganhar) [inline extensions/CocoStudio/Reader/SceneReader.h]: master branch doesn't use c++11, so `enum class` will not be valid.

**Comment by dumganhar** [comment]
[ci rebuild]

**Comment by dumganhar** [comment]
[ci rebuild]

**Pull request merged**